faster_whisper/transcribe.py

```diff
@@ -377,143 +378,9 @@ def load_vad_model(self, vad_onset=0.500, vad_offset=0.363):
-                ):
-                    current_seg_idx += 1
-
-            word_timings.append(_word_timings)
-        return word_timings
-
-    def combine_words(self, metadata, response):
-        combined_segments = []
-
-        for meta, res in zip(metadata, response):
-            word_timestamps = res["word_timestamps"]
-            segment_texts = []
-            segment_index = 0
-            current_segment = meta["segments"][segment_index]
-            current_text = []
-            current_word_timestamps = []
-            current_start = current_segment[0]
-
-            for idx, word_info in enumerate(word_timestamps):
-                word_start, word_end, word_text = (
-                    word_info["start"],
-                    word_info["end"],
-                    word_info["word"],
-                )
-
-                # Move to the next segment if the word is outside the current segment
-                while (
-                    word_start >= current_segment[1]
-                    and segment_index < len(meta["segments"]) - 1
-                ):
-                    # Save the completed segment
-                    if current_text:
-                        segment_texts.append(
-                            {
-                                "start": current_start,
-                                "end": current_segment[1],
-                                "text": "".join(current_text),
-                                "word_timestamps": current_word_timestamps,
-                                "avg_logprob": res["avg_logprob"],
-                                "no_speech_prob": res["no_speech_prob"],
-                            }
-                        )
-                    segment_index += 1
-                    current_segment = meta["segments"][segment_index]
-                    current_start = current_segment[0]
-                    current_text = []
-                    current_word_timestamps = []
-
-                # Add word to the current segment text
-                if word_start >= current_segment[0] and word_end <= current_segment[1]:
-                    current_text.append(word_text)
-                    current_word_timestamps.append(word_info)
-
-            # Save the final segment
-            if current_text:
-                segment_texts.append(
-                    {
-                        "start": current_start,
-                        "end": current_segment[1],
-                        "text": "".join(current_text),
-                        "word_timestamps": current_word_timestamps,
-                        "avg_logprob": res["avg_logprob"],
-                        "no_speech_prob": res["no_speech_prob"],
-                    }
-                )
-
-            combined_segments.extend(segment_texts)
-        return combined_segments
-
     def transcribe(
         self,
-        audio: Union[str, np.ndarray],
+        audio: Union[str, torch.Tensor],
```

Comment:
I think that can still support np.ndarray by using this conversion:
`torch.from_numpy(audio)`

Reply:
Done